subprojects/architecture-test/src/test/java/org/gradle/architecture/test/ArchUnitFixture.java

```diff
@@ -96,6 +96,17 @@ public boolean test(Collection<T> input) {
         };
     }
 
+    static ArchCondition<JavaClass> beAbstract() {
+        return new ArchCondition<JavaClass>("verify that classes are abstract") {
+            @Override
+            public void check(JavaClass input, ConditionEvents events) {
+                boolean fulfilled = input.isInterface() || input.getModifiers().contains(JavaModifier.ABSTRACT);
+                String message = input.getSimpleName() + " must be abstract";
```

Comment:
The message needs to say what the class is. So I think it should be:
- fulfilled: `<className> is abstract`
- not fulfilled: `<className> is not abstract`